thematic_areas/models.py

```diff
@@ -190,6 +191,20 @@ def __str__(self):
             self.level2,
         )
 
+    @classmethod
+    def load(cls, user, thematic_area_data=None):
+        if thematic_area_data or not cls.objects.exists():
```

Comment:
Trocar
`if thematic_area_data or not cls.objects.exists():
`
por 
`if not cls.objects.exists():
`
senão nunca vai executar